spec/services/course_attribute_formatter_service_spec.rb

```diff
@@ -1,4 +1,5 @@
 require_relative "../../app/services/course_attribute_formatter_service"
```

Comment:
You probably don't need this line anymore since you added the one below it

Reply:
We've removed this